libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -469,6 +475,24 @@ public static String decryptBytes(byte[] data, byte[] key, byte[] iv) {
         return null;
     }
 
+    public static String getStringFromStream(InputStream stream) throws IOException {
+        ByteArrayOutputStream output = new ByteArrayOutputStream();
+        byte[] buffer = new byte[READ_BUFFER_LENGTH];
+        int length;
+        while ((length = stream.read(buffer)) != -1) {
+            output.write(buffer, 0, length);
+        }
+        stream.close();
```

Comment:
If an exception is thrown, the stream won't be closed. Put this in a `finally` block.